pkgs/development/python-modules/jupyterlab-git/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, notebook
+, nbdime
+, git
+}:
+
+buildPythonPackage rec {
+  pname = "jupyterlab_git";
+  version = "0.20.0";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qs3wrcils07xlz698xr7giqf9v63n2qb338mlh7wql93rmjg45i";
+  };
+
+  propagatedBuildInputs = [ notebook nbdime git ];
+
+  doCheck = false;
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
They do include checks. However, they didn't run on the NixOS VM I am working on. I'll download from source on my machine and see if the tests work. If they don't, I'll check out the pythonImportsCheck stuff. Thanks!

Reply:
I was able to run all of the tests on my machine (ignoring the async ones) but on the NixOS VM it gave weird errors for the `test_handlers.py` file so I ignored that one. I also added the `pythonImportsCheck` but I can remove that if you think it is unnecessary.